README

```diff
@@ -151,6 +151,32 @@ To create and play with a Test object from the example package,
 	}
 
 
+Parameter interpretation
```

Comment:
Your suggestion is still too narrow. The headline doesn't need to be that specific.

That `parameter` field in `plugin.proto` is exactly what is being used here.